doc/classes/OS.xml

```diff
@@ -430,6 +439,18 @@
 				Not to be confused with [method get_data_dir], which returns the [i]global[/i] (non-project-specific) user home directory.
 			</description>
 		</method>
+		<method name="get_version" qualifiers="const">
+			<return type="String" />
+			<description>
+				Returns the exact production and build version of the operating system. This is different from the branded version used in marketing. This helps to distinguish between different releases of operating systems, including minor versions, and insider and custom builds.
+				For Windows, the major and minor version are returned, as well as the build number. For example, the returned string can look like [code]10.0.9926[/code] for a build of Windows 10, and it can look like [code]6.0.6000[/code] for a build of Windows Vista.
```

Comment:
I'd list a Windows 7 version number instead, as Godot doesn't support Windows Vista (and hasn't been since Godot 3.0 or so).